### PR TITLE
Feature/admin settings/ismail

### PR DESCRIPTION
**Pull request by ialkamal**
## PR TITLE
Admin Settings Migration to New Theme for LAN

**Description:** 
Admin Settings done [Users and Rooms]

  
Migrated admin settings to new theme

  
To integrate the admin settings to main

  
By creating new components

  

### Checklist

 * [x ] My code follows the style guidelines of this project
 * [ x] I have performed a self-review of my own code
 * [x ] My code has been reviewed by at least one peer
 * [x ] I have commented my code, particularly in hard-to-understand areas
 * [x ] I have made corresponding changes to the documentation
 * [x ] My changes generate no new warnings
 * [x ] I have added tests that prove my fix is effective or that my feature works
 * [ x] New and existing unit tests pass locally with my changes
 * [x ] There are no merge conflicts

### CODE REVIEW

**Review by Devon-Brady — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -30,10 +30,13 @@ const App = () => {
...
         <PrivateRoute path="/post/:id" component={Post} />
-        <PrivateRoute path="/room/:id/page/:page" component={SingleRoomDashboard}/>
+        <PrivateRoute
+          path="/room/:id/page/:page"
+          component={SingleRoomDashboard}
+        />
         <PrivateRoute path="/full-search" component={FullSearchResults} />
         <PrivateRoute path="/settings" component={Settings} />
-        <PrivateRoute path="/admin-settings" component={AdminSettings} />
+        {/* <PrivateRoute path="/admin-settings" component={AdminSettings} /> */}
```
Comment (gvicas17) [inline src/App.js:39]: This is commented out do we still need it?

File: src/App.js [hunk 2]
```diff
@@ -30,10 +30,13 @@ const App = () => {
...
         <PrivateRoute path="/mod-settings" component={ModSettings} />
         <PrivateRoute path="/" component={Dashboard} />
         {/* <PrivateRoute path="/faq" component={FaqContent} /> */}
```
Comment (gvicas17) [inline src/App.js:42]: Just noticing another commented out line

**Review by Plofland — APPROVED** [review]
Nice job, Ismail! There was a lot to review but it's all very clean and clear.

File: src/App.js (hunk 1 above)
Comment (gvicas17) [inline src/App.js:39]: This is commented out do we still need it?

**Review by gvicas17 — CHANGES_REQUESTED** [review]
Everything looks great just saw a couple of commented out lines of code that may need to be removed for cleaner codebase.

File: src/App.js (hunk 2 above)
Comment (ialkamal) [reply src/App.js:42]: No we don't, but I would prefer to refactor the App.js file at the end.

File: src/App.js (hunk 1 above)
Comment (ialkamal) [reply src/App.js:39]: No we don't but I'm leaving App.js till the end after we finished the migration of all components. We would need to restructure the components and delete the old components.

**Pull request merged**